### PR TITLE
PROTOTYPE Adding profile page to enterprise learner portal.

### PR DESCRIPTION
**Pull request by douglashall**
**Notes**
1. Uses locally installed version of frontend-app-profile branch: https://github.com/edx/frontend-app-profile/pull/225
2. Uses locally installed version of frontend-analytics: https://github.com/edx/frontend-analytics/pull/8
3. Uses locally installed version of frontend-i18n: https://github.com/edx/frontend-i18n/pull/48
4. Uses client-side routing with reach-router. See `match-path`: https://github.com/edx/frontend-app-learner-portal/pull/143/files#diff-aa13cdc897236377de5ef7ac15611739R58
5. Image assets copied over from frontend-app-profile to get sass compilation to work.

### CODE REVIEW

File: src/components/enterprise/App.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const App = (props) => {
+  const { pageContext } = props;
+  const { enterpriseName } = pageContext;
+  return (
+    <>
+      <Helmet title={enterpriseName} />
+      <Router>
+        <Layout path="/" pageContext={pageContext}>
+          <DashboardPage path="/" pageContext={pageContext} />
```
Comment (adamstankiewicz) [inline src/components/enterprise/App.jsx]: If we're already passing `pageContext` into `Layout`, `pageContext` data should be accessible via the context API within `DashboardPage` and probably don't need to pass it as a prop here.

File: src/components/common/layout/Layout.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { IntlProvider } from 'react-intl';
 import SiteHeader from '@edx/frontend-component-site-header';
 import SiteFooter from '@edx/frontend-component-footer';
 import { connect } from 'react-redux';
+import { Link } from "@reach/router";
```
Comment (adamstankiewicz) [inline src/components/common/layout/Layout.jsx]: Out of curiosity, did you try or consider using Gatsby's router that wraps @reach/router? The benefit of using Gatsby's wrapper around @reach/router is that you get prefetching when switching between routes (https://www.gatsbyjs.org/docs/gatsby-link/).

**Comment by schenedx** [comment]
Definitely out of date.

**Comment by douglashall** [comment]
@schenedx Come on man! This was good stuff. :)

**Pull request closed without merging**